src/utils/configurationDataHandler.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */
+export const handleEtrustedInteraction = (
+  token: string | undefined,
+  allState: any,
+  interactionType: string,
+  callback: (token: string, payload: any) => Promise<void>
+): void => {
+  if (!token) {
+    console.error('Token is not available')
+    return
+  }
+
+  try {
+    callback(token, { action: interactionType, allState }).catch(error => {
+      console.error(`Error during ${callback.name}:`, error)
+    })
+  } catch (error) {
+    console.error(`Error during ${callback.name} invocation:`, error)
+  }
+}
+
+export const handleEtrustedConfiguration = (
+  token: string | undefined,
+  allState: any,
+  callback: (token: string, payload: any) => Promise<void>
```

Comment:
If you don't mind about the payload type then it's always better to use unknown and you don't need to disable eslint rule ;)